nucypher/utilities/porter/porter.py

```diff
@@ -69,6 +107,80 @@ def __init__(self,
             self.make_cli_controller()
         self.log.info(self.BANNER)
 
+    def get_treasure_map(self, map_identifier: str, bob_encrypting_key: UmbralPublicKey, timeout=3):
```

Comment:
Isn't `map_identifier` a `bytes` object? Also, perhaps I am being too picky, but it worries me that we're passing around plain byte objects (map ID here, or `treasure_map_bytes` below). Feels like it can be a source of errors. Ideally we should serialize at the latest possible moment.

Reply:
My bad, it is in fact a string. Still it would be better off at least as a type alias (like `ChecksumAddress`).

Reply:
Same comment for `MapID` in your other comment (https://github.com/nucypher/nucypher/pull/2717#discussion_r651036501). If we are going to do it, we should address it in all areas.